datasetinsights/evaluation_metrics/average_precision_2d.py

```diff
@@ -1,6 +1,6 @@
 r"""Reference.
 
-https://github.com/rafaelpadilla/Object-Detection-Metrics#average-precision\
+https://github.com/rafaelpadilla/Object-Detection-Metrics#average-precision\.
```

Comment:
??? Why add extra dot here?  Can you improve the docstring of this reference section?

Reply:
Will improve it.